tests/cypress/support/commands.js

```diff
@@ -252,6 +252,14 @@ Cypress.Commands.add(
     }
 )
 
+/**
+ * Wait until the map has been rendered and is ready. This is useful and needed during the
+ * application startup phase
+ */
+Cypress.Commands.add('waitMapIsReady', ({ timeout = 15000 } = {}) => {
+    cy.waitUntilState((state) => state.app.isMapReady, { timeout: timeout })
+})
```

Comment:
It might make sense to use this in the ubiquitous `.goToMapView` command, don't you think?

Reply:
It is already Part of it